survey_goals_and_questions.md

```diff
@@ -0,0 +1,169 @@
+The draft survey itself.
+
+# Survey length
+Based on feedback from the community, a ~10 minute survey is ideal and would maximize participation likelihood.  Beyond length, we will make most questions optional so that participants can pick and choose questions if they are rushed or prefer not to answer.  Progress will be shown throughout.
+
+# Survey Topics
+Based on discussions with many in the community, the below topics were most frequently mentioned as items folks hoped to learn about the community / were perceived as most valuable to learn about the community.
+
+Each topic has:
+* Estimated timing
+* What we want to learn
+* Outcomes we hope to achieve by learning
+* List of questions to be asked
+
+# Who we are: 5 mins
+## [2 mins] Basic demographics of person & company 
+### What do we learn / get by asking this?
+* Get a baseline view of the types of orgs that are applying MLOps
+* Get a baseline view of how ML teams are structured & who they work with
+* Use as filters to analyze other questions
+### Questions
+* What is your current job title?
+* What bucket does your role fall in?
```

Comment:
maybe we need to put other titles like head of data, CDO, CTO etc. I am wondering if its worth having this or just using the above question about what the persons title is

Reply:
What about something like this?

```suggestion
* Which best describes you?
    * Executive or C-suite
    * Director (leader of leaders)
    * Manager of a team
    * Individual contributor
    * Other (please describe)
* [FOR IC ONLY] What bucket does your role fall in?
* [FOR MGR+] Which of the following roles are on your team?
    * Other (please describe)
```